app/authorization.go

```diff
@@ -80,7 +80,7 @@ func (a *App) SessionHasPermissionToChannel(session model.Session, channelId str
 		return true
```

Comment:
@mkraft now you see the context of my DM earlier :)

I left this where it is, but I was wondering about it because it feels like the flow of this function could be cleaned up a bit. However, I note moving this condition earlier in the function *would* have a behaviour change - meaning a "not found" on the channel would return `true` rather than `false` for unrestricted sessions - so I thought it would be better to leave any such change out of this specific fix.